### PR TITLE
Adding Index links to top of page - Common Actions

### PR DESCRIPTION
**Pull request by claycrenshaw**
Closes #

{{short description}}

#### Changelog

**New**

- {{new thing}}

**Changed**

- {{change thing}}

**Removed**

- {{removed thing}}

### CODE REVIEW

File: src/content/patterns/common-actions/common-actions.md [hunk 1]
```diff
@@ -4,6 +4,28 @@ title: Common Actions
 
 **Common actions** are frequently used actions that appear multiple times across different components and workflows. For platform consistency, these actions should only be applied in the ways described below.
 
+<b>Regressive Actions</b>
```
Comment (alisonjoseph) [inline src/content/patterns/common-actions/common-actions.md]:
```suggestion
<strong>Regressive Actions</strong>
```

File: src/content/patterns/common-actions/common-actions.md [hunk 2]
```diff
@@ -4,6 +4,28 @@ title: Common Actions
...
+<ul>
+    <li><a href="#cancel">Cancel</a></li>
+    <li><a href="#clear">Clear</a></li>
+    <li><a href="#close">Close</a></li>
+    <li><a href="#delete">Delete</a></li>
+    <li><a href="#remove">Remove</a></li>
+    <li><a href="#reset">Reset</a></li>
+</ul>
+</anchor-links>
+<b>Progressive Actions</b>
```
Comment (alisonjoseph) [inline src/content/patterns/common-actions/common-actions.md]:
```suggestion
<strong>Progressive Actions</strong>
```

**Comment by lovemecomputer** [comment]
@alisonjoseph @claycrenshaw as part of another PR, i created an option onto anchor links: the property `small`

so for example, on the **Guidelines > Layout**, several sub-sections have their own set of index links that are smaller to match Jean's spec.

You can use this smaller by setting `small` to `true` when using the `anchor-links` component!

**ex:**
```
<anchor-links small="true">
... 
</anchor-links>
```

these were designed to be used throughout the document sections, hopefully it will work alright for this use case! if not maybe we'll need a design consult

**Comment by alisonjoseph** [comment]
👍 merging in, but we probably need a design issue for this page https://deploy-preview-687--carbon-website.netlify.com/patterns/common-actions . cc @carbon-design-system/design

**Pull request merged**